tests/resources/primary-podify-demo.yaml

```diff
@@ -0,0 +1,38 @@
+# Save the output of this file and use kubectl create -f to import
```

Comment:
Probably a license might be here? But I am not sure. It is generated from podman-desktop using some podman feature. it is a question then...

Reply:
Which or from where would I get that license?

Reply:
basically any file in this codebase should have it. But that is more of a question I am not sure about the answer. :)
We can probably keep what we got now there.

Reply:
ok, left that part unchanged.